modules/large-deployments/pages/hub-install.adoc

```diff
@@ -1,14 +1,27 @@
 [[lsd-hub-install]]
 = Hub Installation
 
-Before you begin, you need to install the XMLRPC API package, and configure the Hub Server to use the API.
+Before you begin, you need to install the ``hub-xmlrpc-api`` package, and configure the Hub Server to use the API.
 
 
 
-.Procedure: Installing and Configuring the XMLRPC API
+.Procedure: Installing and Configuring the Hub XMLRPC API
 . On the Hub Server, or on a host that has access to all peripheral servers' XMLRPC APIs, install the ``hub-xmlrpc-api`` package.
 The package is available in the {productname} {productnumber} repositories.
-. Verify if the configuration file available at  ``/etc/hub/hub-xmlrpc-api-config.json`` is pointing to the correct Hub instance and modify it as required.
-. OPTIONAL: In the configuration file, check the settings for the timeout parameters ``connect_timeout`` and ``read_write_timeout``.
-The values for these parameters are defined in seconds.
-. Restart services to pick up the configuration changes.
+. OPTIONAL: the Hub XMLRPC API service does not start on installation. You can set it up to start automatically with systemd, by executing the following commands
+---
+sudo systemctl enable hub-xmlrpc-api.service
+sudo systemctl start hub-xmlrpc-api.service
+---
+. OPTIONAL: verify contents of the configuration file at  ``/etc/hub/hub.conf``
+  - ``HUB_API_URL``: URL to the Hub Server XMLRPC API endpoint. The default is correct if you are installing `hub-xmlrpc-api` on the Hub Server
+  - ``HUB_CONNECT_TIMEOUT``: maximum number of seconds to wait for a response when connecting to a Server. In most cases, these parameter can be left at its default value.
+  - ``HUB_REQUEST_TIMEOUT``: maximum number of seconds to wait for a response when calling a Server method. In most cases, these parameter can be left at its default value.
+  - ``HUB_CONNECT_USING_SSL``: use https instead of plain http for communicating with peripheral Servers. Recommended for a secure environment.
```

Comment:
```suggestion
*  ``HUB_API_URL``: URL to the Hub Server XMLRPC API endpoint. 
Use the default value if you are installing `hub-xmlrpc-api` on the Hub Server.
* ``HUB_CONNECT_TIMEOUT``: the maximum number of seconds to wait for a response when connecting to a Server. 
Use the default value in most cases.
* ``HUB_REQUEST_TIMEOUT``: the maximum number of seconds to wait for a response when calling a Server method. 
Use the default value in most cases.
* ``HUB_CONNECT_USING_SSL``: use HTTPS instead of HTTP for communicating with peripheral Servers. 
Recommended for a secure environment.
```